src/virtualbox-template.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0"?>
+<Envelope xmlns="http://schemas.dmtf.org/ovf/envelope/1" xmlns:cim="http://schemas.dmtf.org/wbem/wscim/1/common" xmlns:ovf="http://schemas.dmtf.org/ovf/envelope/1" xmlns:rasd="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_ResourceAllocationSettingData" xmlns:vssd="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_VirtualSystemSettingData" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:vbox="http://www.virtualbox.org/ovf/machine">
+  <References>
+    <File ovf:href="disk.vmdk" ovf:id="file1" ovf:size="{vmdk_size}"/>
+  </References>
+  <DiskSection>
+    <Info>Virtual disk information</Info>
+    <Disk ovf:capacity="{vmdk_capacity}" ovf:capacityAllocationUnits="byte" ovf:diskId="vmdisk1" ovf:fileRef="file1" ovf:format="http://www.vmware.com/interfaces/specifications/vmdk.html#streamOptimized" ovf:populatedSize="{vmdk_size}" vbox:uuid="c7b8d618-f480-433e-b0c4-07b0d69d4918"/>
```

Comment:
With the hardcoded UUID, are there conflicts if the same OVA is imported multiple times?

Reply:
It might make sense to go back to generating a distinct UUID on each build, to help distinguish between different images.  Not sure though.

Reply:
Unfortunately the GUID seems to be required; however, it does not seem to be a conflict after import. I was able to import many times over with no issue.